tensorflow/stream_executor/cuda/cuda_stub.cc

```diff
@@ -96,4 +96,8 @@ typedef void(CUDA_CB* CUhostFn)(void* userData);
 // We only need one stub implementation. Calling a function that is not yet
 // available in the given CUDA version will return
 // CUDA_ERROR_SHARED_OBJECT_INIT_FAILED.
+#if CUDA_VERSION < 10010
```

Comment:
The comment doesn't match the implementation anymore.

Reply:
Fixed.